be/src/agent/task_worker_pool.cpp

```diff
@@ -1911,9 +1911,10 @@ void CloneTaskPool::_clone_worker_thread_callback() {
             LOG_INFO("successfully clone tablet")
                     .tag("signature", agent_task_req.signature)
                     .tag("tablet_id", clone_req.tablet_id);
+            ++_s_report_version;
```

Comment:
line 1914,   1917： only clone a not exisist replica  can  add s_report_version and report s_report_version.

Because for clone a existsing replica,  the report tablet  no change.

Reply:
只有在clone 新副本时，增加_s_report_version和上报_s_report_version。

对于已有的副本clone时，不要修改_s_report_version也不要上报它。 因为对一个已存在的tablet 增量clone，不会出现issue所说的问题。 

假如增量clone时增大_s_report_version时，那可能导致fe丢弃上一个BE tablet report（旧report version）。这会有问题：假如BE很多增量clone， 那FE会频繁丢弃这个BE 的tablet report。